ui/src/components/DashboardPlane.js

```diff
@@ -44,13 +43,15 @@ const DashboardPlane = () => {
             label={intl.formatMessage({ id: 'control_plane' })}
             status={planesStatus}
             alerts={planesHighestSecurityAlert}
+            showArrow={false}
```

Comment:
I checked with @Cuervino, so I think you were right, let's remove the arrow here.